src/terra-dev-site/contributing/DeveloperWorkflow.c.contributing.md

```diff
@@ -0,0 +1,171 @@
+
+## Install Dependencies
+
+After you’ve gone through the [git workflow](#/contributing/terra-ui/git-workflow) and created your branch, the next step will be to install the project dependencies.
+
+Run the following command to install the npm dependencies.
+
+`npm install`
+
+Make sure you’re using the correct NodeJS version (we recommend using [nvm](https://github.com/creationix/nvm) to install and manage node on your machine)
+
+Run the following command to switch the node version required for the project.
+
+`nvm use`
+
+If your terminal tells you that you don’t have that version installed, run
+
+`nvm install`
+
+## Starting The Development Server
+
+Running `npm start` will start up the development server. It will be available at http://localhost:8080/
+
+You can navigate to different component documentation pages. These pages are auto-generated via files in the  [terra-dev-site](https://github.com/cerner/terra-dev-site) doc directories found in each package directory.
+
+## Starting The Development Server For Internet Explorer 10
+
+// TODO
+
+## Follow Component Standards
+As you work on your code changes, ensure the components follow [the Terra UI component standards](https://github.com/cerner/terra-core/wiki/Component-Features).
+
+## Running Lint Scripts
+For Terra UI projects, we’ve created our own [ESLint config](https://github.com/cerner/eslint-config-terra) and [Stylelint config](https://github.com/cerner/stylelint-config-terra).
+
+* Our ESLint config is based on [Airbnb’s JavaScript style guide](https://github.com/airbnb/javascript).
+* Our Stylelint config is based on [sass-guidelin.es](https://sass-guidelin.es/).
+
+Inside the monorepos, running the following command in the root directory to run both ESLint and Stylelint for all packages.
+
+`npm run lint`
+
+* You can also run `npm run lint:js` and this will only run ESLint.
+* You can also run `npm run lint:scss` and this will only run StyleLint.
+
+Along with being able to run these scripts across the entire project, you can run these scripts in individual package directories.
+
+Change directories to the specific package in your terminal, e.g. `cd packages/terra-button`, and run the same lint commands for ESLint and Stylelint.
+
+```
+npm run lint
+npm run lint:js
+npm run lint:scss
+```
+
+We recommend that you install a plugin to your editor to run [ESLint](https://eslint.org/docs/user-guide/integrations#editors) and [Stylelint](https://stylelint.io/user-guide/complementary-tools/#editor-plugins) automatically which will then show you any errors inline.
+
+## Testing Components
+
+Terra UI has 2 types of tests:
+* Unit tests, using Jest
+* Browser tests, using Webdriver.io
+
+### Jest
+We use Jest for testing, including unit tests for functionality and snapshot testing for components.
+
+If you make intentional changes to an existing component, you will need to update its snapshot:
+npm test -- -u
```

Comment:
```bash
npm test -- -u
` ` `

Reply:
These might be of interest:
- https://www.codementor.io/pkodmad/dom-testing-react-application-jest-k4ll4f8sd
- https://github.com/auth0/blog/blob/master/_posts/2017-01-26-testing-react-applications-with-jest.markdown